src/pages/user/home/index.jsx

```diff
@@ -1,32 +1,45 @@
 import { Box, Container, Typography } from '@mui/material';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 
 import Header from '../../../components/headerClient/Header';
 import { SliderComp } from '../../../components';
 import SwiperComp from '../../../components/swiper';
 
-const images = ['https://betterbody.ie/wp-content/uploads/elementor/thumbs/strong-man-training-in-gym-1-scaled-puc6dnltxn54i7tqe8yfryh0a4wn9r8z3ewgicqsqo.jpg',
-  'https://betterbody.ie/wp-content/uploads/elementor/thumbs/strong-man-training-in-gym-1-scaled-puc6dnltxn54i7tqe8yfryh0a4wn9r8z3ewgicqsqo.jpg',
-  'https://betterbody.ie/wp-content/uploads/elementor/thumbs/strong-man-training-in-gym-1-scaled-puc6dnltxn54i7tqe8yfryh0a4wn9r8z3ewgicqsqo.jpg',
-  'https://betterbody.ie/wp-content/uploads/elementor/thumbs/strong-man-training-in-gym-1-scaled-puc6dnltxn54i7tqe8yfryh0a4wn9r8z3ewgicqsqo.jpg',
-  'https://betterbody.ie/wp-content/uploads/elementor/thumbs/strong-man-training-in-gym-1-scaled-puc6dnltxn54i7tqe8yfryh0a4wn9r8z3ewgicqsqo.jpg',
-  'https://betterbody.ie/wp-content/uploads/elementor/thumbs/strong-man-training-in-gym-1-scaled-puc6dnltxn54i7tqe8yfryh0a4wn9r8z3ewgicqsqo.jpg'];
+const Home = () => {
+  const [trainersData, setTrainersData] = useState([]);
 
-const Home = () => (
-  <Box>
-    <Header />
-    <Container>
-      <Box>
-        <Box mb={10}>
-          <SliderComp />
-        </Box>
-        <Box mb={10} position="relative">
-          <SwiperComp images={images} />
+  const handleDataChange = (newData) => {
+    setTrainersData(newData);
+  };
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const trainers = await axios.get('http://localhost:5050/api/users/trainers');

```

Comment:
we don't add the full URL, it's already declared in the vite config file. add only the endpoint